mods/ts/rules/palettes.yaml

```diff
@@ -22,7 +22,6 @@
 		Name: chrome
 		Filename: cameo.pal
 		AllowModifiers: false
-		ShadowIndex: 242
```

Comment:
Damn...

Anyway, the problem here is that Westwood did not really use regular shadow, but rather translucency, and iirc TS clock uses `anim.pal` rather than `cameo.pal`, too. So I guess the correct solution will be to add a `ClockPalette` to `Buildable`.